src/app/pages/orders/components/orders/orders.component.html

```diff
@@ -0,0 +1,17 @@
+<ng-container *ngIf="orders; else loading">
+  <div class="flex flex-col md:flex-row items-center justify-center w-full flex-wrap order-1 md:order-0 pb-48 md:pb-36">
+    <div class="hidden md:flex md:w-10/12 justify-evenly items-center h-16 bg-grey-200 sticky top-0 rounded-md">
+      <span class="font-bold text-center">{{ buildTranslationKey('orderId') | translate }}</span>
+      <span class="font-bold text-center">{{ buildTranslationKey('products') | translate }}</span>
+      <span class="font-bold text-center">{{ buildTranslationKey('total') | translate }}</span>
+      <span class="font-bold text-center">{{ buildTranslationKey('status') | translate }}</span>
+      <span class="font-bold text-center">{{ buildTranslationKey('dateCreated') | translate }}</span>
```

Comment:
There is 5 line, which are almost the same, won't be better with an array and a loop?

Reply:
@WeszeRita , you're right, but it's not so critical in this case :)

Reply:
I refactored it with the array method :)